paywall/src/__tests__/unlock.js/Wallet/setupUserAccounts.test.ts

```diff
@@ -26,36 +24,62 @@ describe('Wallet.setupUserAccounts()', () => {
     },
     unlockUserAccounts: true,
   }
-  const startup: StartupConstants = {
-    network: 1984,
-    debug: 0,
-    paywallUrl: 'http://paywall',
-    accountsUrl: 'http://app/accounts',
-    erc20ContractAddress: '0x591AD9066603f5499d12fF4bC207e2f577448c46',
-  }
 
-  function makeWallet() {
+  let setUserAccountAddress = jest.fn()
+  let setUserAccountNetwork = jest.fn()
+  let createIframeSpy: any
+
+  beforeAll(() => {
```

Comment:
One notable change here is that we use the same instance of the window and iframes for all of these tests. That makes it a little more straightforward to read.